test/integ/test_session_parameters.py

```diff
@@ -115,3 +115,70 @@ def set_backend_client_session_keep_alive(
         db_parameters["account"], db_parameters["user"], str(val)
     )
     admin_cnx.cursor().execute(query)
+
+
+@pytest.mark.skipif(
+    not CONNECTION_PARAMETERS_ADMIN,
+    reason="Snowflake admin required to setup parameter.",
+)
+def test_htap_optimizations(db_parameters: object) -> None:
+    # alter account ENABLE_SNOW_654741_FOR_TESTING=true
+    admin_cnx = snowflake.connector.connect(
+        protocol=db_parameters["sf_protocol"],
+        account=db_parameters["sf_account"],
+        user=db_parameters["sf_user"],
+        password=db_parameters["sf_password"],
+        host=db_parameters["sf_host"],
+        port=db_parameters["sf_port"],
+        warehouse=db_parameters["sf_warehouse"],
+        database=db_parameters["sf_database"],
+        schema=db_parameters["sf_schema"],
+    )
+    try:
+        query = "alter account {} set ENABLE_SNOW_654741_FOR_TESTING=true".format(
+            db_parameters["sf_account"]
+        )
```

Comment:
Might be too long, but please use f-strings instead of format (unless you want to insert the same things twice)
```suggestion
        query = f"alter account {db_parameters["sf_account"]} set ENABLE_SNOW_654741_FOR_TESTING=true"
```